lib/collection/src/wal_delta.rs

```diff
@@ -14,8 +14,12 @@ pub type LockedWal = Arc<ParkingMutex<SerdeWal<OperationWithClockTag>>>;
 /// A WAL that is recoverable, with operations having clock tags and a corresponding clock map.
 pub struct RecoverableWal {
     pub(super) wal: LockedWal,
+
     /// Map of all highest seen clocks for each peer and clock ID.
+    //
+    // TODO: Rename to `newest_clocks` (or `newest_*` something)?
```

Comment:
Sure! I see that you've also used _new_ in other places, so let's follow that here then.

Please also update the file name to `clock_map_newest.json`.